pkg/kapis/monitoring/v1alpha3/register.go

```diff
@@ -555,6 +557,16 @@ func AddToContainer(c *restful.Container, k8sClient kubernetes.Interface, monito
 		Returns(http.StatusOK, respOK, monitoringdashboardv1alpha2.ClusterDashboard{})).
 		Produces(restful.MIME_JSON)
 
+	ws.Route(ws.POST("/namespaces/{namespace}/dashboards/{grafanaDashboardName}/template").
+		To(h.handleGrafanaDashboardImport).
+		Doc("Convert Grafana templates to KubeSphere clusterdashboards.").
```

Comment:
`clusterdashboards` in the doc should be renamed to `dashboards`

Reply:
will be fixed